Automattic/meeting-calendar
# Tools: Add wp-env, phpunit, phpcs

This PR adds a wp-env environment along with composer to install phpunit and phpcs, so that the unit tests can be run locally and through a github action. Now all PRs and pushes to master will run the phpunit tests. This PR also fixes the auto-fixable phpcs issues, but there are still a number of other issues so the linter was not added to the GH workflow.

To test:

1. Check that the instructions in the readme make sense
2. Update & install the new dependencies
    `npm install`
    `composer install`
3. Start the environment `npm run wp-env start`
4. Run the tests locally `npm run test:unit-php`
5. They'll fail, but another PR (in process) will fix them

Or, watch the workflow on this PR.
